### ISSUE TITLE
Refreshing branches not working in beta 10

### ISSUE THREAD

**Issue by jdixon567** [opendevshop/devshop#144]
Steps to reproduce:

1) Create a project
2) Create first environment in project during project creation (this actually refreshes branches for me, so a workaround is deleting a project and creating it again)
3) Create a new branch in the repository and push to your repo
4) Try to create a second environment in project created in step 2.  Branch from step 3 is not listed
5) Hover over project repo branches, and click "refresh branches"
6) Branches are not updated
7) Try to create a second environment in project created in step 2 again.  Branch from step 3 is still not listed.

Not sure if this is happening for everyone.  I'm having this issue in Ubuntu 14.04 with devshop beta 10 and the git repo I'm connecting to is on bitbucket.

**Comment by jonpugh** [comment]
> 4) Try to create a second environment in project created in step 2. Branch from step 3 is not listed

If this happens before the "Project Verify" task completes, you won't see the new branch. Wait for the task to complete.

> 5) Hover over project repo branches, and click "refresh branches"
> 6) Branches are not updated

I just fixed this in https://github.com/opendevshop/devmaster/commit/2b47119576a1bd466e0175d2172a1063a5292ec1

Is it possible your new branches have slashes in them? Up until today, branches with slashes did not work.  I fixed that in https://github.com/opendevshop/devmaster/commit/389019a9b4314d50bbc04683abaace25f3a5d0ba
